AE_navigation_guide.ipynb

```diff
@@ -22,7 +22,7 @@
     "2) Transforming the data or conducting an analysis\n",
     "3) Exporting data or visualizations\n",
     "\n",
-    "You can also access these notebooks through the file directory on the left side of the screen. These notebook files will automatically update to reflect our latest development, so we suggest creating a copy of the file if you want to change or insert your own code. Each notebook is flexibly designed so that users can follow existing analyses informed by best practices in climate science and build on or modify these analyses to meet their own specific needs."
+    "You can also access these notebooks through the file directory on the left side of the screen. These notebook files will automatically update to reflect our latest development, so we suggest creating a copy of the file if you want to change or insert your own code. In addition, your own notebooks will stay available and secure in the file directory whether you log in or out. Each notebook is flexibly designed so that users can follow existing analyses informed by best practices in climate science and build on or modify these analyses to meet their own specific needs."
```

Comment:
Suggest "Your copies of notebooks will persist in your personal file directory between sessions."